e2e/templates/cni-install.yml.j2

```diff
@@ -7,7 +7,7 @@ metadata:
 data:
   install_cni.sh: |
     cd /tmp
-    wget https://github.com/containernetworking/plugins/releases/download/v1.1.1/cni-plugins-linux-amd64-v1.1.1.tgz
+    wget https://github.com/containernetworking/plugins/releases/download/v1.4.0/cni-plugins-linux-amd64-v1.4.0.tgz
     cd /host/opt/cni/bin
     tar xvfzp /tmp/cni-plugins-linux-amd64-v1.1.1.tgz
```

Comment:
you missed this cni-plugins-linux-amd64-v1.1.1.tgz one

Reply:
Good catch! Fixed.